languages/swift/exercises/concept/closures/.docs/hints.md

```diff
@@ -0,0 +1,32 @@
+## General
+
+- The syntax for defining a closure is `{ (parameter list) -> ReturnType in bodyOfClosure }`
```

Comment:
Instead of listing the syntax, is there perhaps an external resource you can refer to?